portfolio-alex.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+
+<html>
+	
+	<head>
+		
+		<link rel="stylesheet" type="text/css" href="portfolio-alex.css"/>
+		<link href="https://fonts.googleapis.com/css?family=Spartan&display=swap" rel="stylesheet"/>
+		
+		<meta name="viewport" content="width=device-width,initial-scales=1"/>
+		
+		<title>Alex Jennings Portfolio</title>
+		
+	</head>	
+	
+		<body>
```

Comment:
Your indentation is a bit messy throughout this, might be worth cleaning it up so that it is easier to read. :)